graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/scriptingapi/ScriptingApiResource.java

```diff
@@ -167,7 +170,13 @@ public TabularResponse executeQuery(@ApiParam(name = "query") @QueryParam("query
                                         @ApiParam(name = "metrics") @QueryParam("metrics") List<String> metrics,
                                         @Context SearchUser searchUser) {
         try {
-            AggregationRequestSpec aggregationRequestSpec = queryParamsToFullRequestSpecificationMapper.simpleQueryParamsToFullRequestSpecification(query, streams, timerangeKeyword, groups, metrics);
+            AggregationRequestSpec aggregationRequestSpec = queryParamsToFullRequestSpecificationMapper.simpleQueryParamsToFullRequestSpecification(
+                    query,
+                    StringUtils.splitByComma(streams),
+                    timerangeKeyword,
+                    groups,
+                    splitByComma(metrics)
```

Comment:
Please use the same variant throughout the code, either with static import everywhere, or without it.

Reply:
Fixed